src/entities/transactions/transaction.ts

```diff
@@ -0,0 +1,34 @@
+import { ProtocolType } from '../protocolTypes';
+import { TransactionStatus } from './transactionStatus';
+import { TransactionType } from './transactionType';
+
+export interface RainbowTransaction {
+  address?: string;
+  balance: {
+    amount: string;
+    display: string;
+  };
+  dappName?: string;
+  description: string | null;
+  from: string | null;
```

Comment:
We have a `EthereumAddress` type on the wallet. Maybe we should move that somewhere and start using it here...

Reply:
Done